src/backend/serial/curve_models/mod.rs

```diff
@@ -182,6 +184,14 @@ pub struct AffineNielsPoint {
     pub xy2d:      FieldElement,
 }
 
+impl Zeroize for AffineNielsPoint {
+    fn zeroize(&mut self) {
+        self.y_plus_x.zeroize();
+        self.y_minus_x.zeroize();
+        self.xy2d.zeroize();
```

Comment:
Alternatively this could be implemented using a proc macro with `zeroize_derive`, if you'd prefer.